serverlessworkflow/modules/ROOT/pages/service-orchestration/orchestration-of-grpc-services.adoc

```diff
@@ -1 +1,164 @@
+Therefore, in this case, the flow declares that `greeting.proto` is expected to be found in the project classpath, that the service to be used is `Greeter` and the method name is `sayHello` 
+
+The flow execution consists of a single link:https://github.com/serverlessworkflow/specification/blob/main/specification.md#operation-state[operation] state, composed of one https://github.com/serverlessworkflow/specification/blob/main/specification.md#action-definition[action].
+
+[source,json]
+----
+ {
+    "name": "sayHello",
+    "functionRef" : {
+        "refName": "sayHello",
+        "arguments": {
+            "name": ".name",
+            "language": ".language"
+        }
+    }
+}
+----
+
+The action invokes the `sayHello` RPC method using as parameters two model variables: `name` and `language`, provided as part of the REST call that starts the flow. 
+ 
+=== Application properties
+
+We need to tell the application where the gRPC server is located.
+
+[source,properties]
+----
+quarkus.grpc.clients.Greeter.host=localhost
+quarkus.grpc.clients.Greeter.port=50051
+----
+
+Internally, the Kogito Serverless Workflow implementation uses the same set of properties as link:https://quarkus.io/guides/grpc-service-consumption#client-configuration[Quarkus gRPC client].
+The "client-name" used by Kogito is the service name declared in the proto file. `Greeter` in this case.
+
+Why is port 50051? In the same repo as the workflow application, a link:https://github.com/kiegroup/kogito-examples/tree/main/kogito-quarkus-examples/serverless-workflow-greeting-rpc-quarkus/serverless-workflow-greeting-server-rpc-quarkus[maven project] is available, which provides a simple implementation of the greeter service. By default, this gRPC server runs on port 50051, hence the value for the port property. 
+
+=== Maven POM 
+
+gRPC functionality is built within link:https://github.com/kiegroup/kogito-runtimes/tree/main/quarkus/extensions/kogito-quarkus-serverless-workflow-extension[Serverless Workflow Quarkus extension], which means that there is no need to include specific dependencies for gRPC to work with Serverless Workflow. 
+The same set of dependencies included for the Hello Workflow example will work. 
+
+One section of the pom.xml file that deserves mention is this one:
+
+[source,xml]
+----
+<resource>
+ <directory>${project.basedir}/../serverless-workflow-greeting-server-rpc-quarkus/src/main/proto</directory>
+ <includes>
+   <include>greeting.proto</include>
+  </includes>
+</resource>
+----
+
+Remember that the proto file, because the URI specified for it does not have any scheme, is expected to be accessible in the project's classpath. Therefore, this snippet ensures the proto file defined in the gRPC server project is included in the classpath of the workflow application. 
```

Comment:
```suggestion
Remember that the proto file, because the URI specified for it does not have any scheme, is expected to be accessible in the project's classpath. Therefore, this snippet ensures the proto file defined in the gRPC server project is included in the workflow application's classpath. 
```